src/Profile_User.php

```diff
@@ -379,6 +379,7 @@ public static function showForEntity(Entity $entity)
             );
 
             $current_pid = null;
+            $i = 0;
```

Comment:
I guess that change the behavior... Previousely, it was set to 0 on condition.

Reply:
commit replaced by 5b78074963b215ead6e7e736728cb3c4d1db38ac

On first iteration, `if ($data['pid'] != $current_pid) {` is always false, so `$i` value will be anyway set to `0`.